Lesson26/Gokhan/script.js

```diff
@@ -0,0 +1,57 @@
+// replace with your own API KEY (register on unsplash.com to get it).
+
+// Lesson 26 My API_KEYUcxmetuPnt2wd3vltsdXqYaOSvDxirnsB7rh5iRrBHI
+// Lesson 26 API_KEY qCnxm3oi9NS9mzAkGmh7_4B6V3p6DJnyFs_wKfNQFF4 Lesson 26 API_KEY
+
+const API_KEY = 'UcxmetuPnt2wd3vltsdXqYaOSvDxirnsB7rh5iRrBHI';
+
+const fetchButton = document.getElementById('fetch-button');
+const imageCountSelect = document.getElementById('image-count');
+const imageContainer = document.getElementById('image-container');
+
+fetchButton.addEventListener('click', () => {
+
+	const count = imageCountSelect.value;
+
+	fetch(`https://api.unsplash.com/photos/random?count=${count}&client_id=${API_KEY}`)
+		.then((response) => response.json())
+		.then((data) => {
+			imageContainer.innerHTML = '';
+			data.map((imgData) => {
+				const newImage = document.createElement(`img`);
+				newImage.src = imgData.urls.regular;
+				newImage.alt = imgData.alt_description;
+				newImage.classList.add('image');
+				imageContainer.appendChild(newImage);
+			});
+		})
+		.catch((error) => console.log(error));
+});
+
+const resetButton = document.getElementById('reset-button');
+resetButton.style.display = 'none';
```

Comment:
It's generally a good practice to separate concerns by managing styles in CSS rather than directly in JavaScript. This approach enhances maintainability and readability. Instead of setting styles directly in the JavaScript code, consider defining CSS classes for the styles you want to apply and then toggling these classes using JavaScript. Here's how you can refactor the code:
```
.hidden {
  display: none;
}

.centered {
  margin: 0 auto;
}
```

Update JavaScript:
```
const resetButton = document.getElementById('reset-button');
resetButton.classList.add('hidden');

fetchButton.addEventListener('click', () => {
  resetButton.classList.remove('hidden');
  resetButton.classList.add('centered');
});

resetButton.addEventListener('click', () => {
  imageContainer.innerHTML = '';
  resetButton.classList.add('hidden');
});
```

This way, you maintain a clear separation between the styling and the behavior of your elements, making it easier to manage styles across your application.